xmtp_mls/src/groups/mod.rs

```diff
@@ -2955,8 +2950,7 @@ pub(crate) mod tests {
         assert_eq!(alix_groups.len(), 1);
     }
 
-    #[cfg(not(target_arch = "wasm32"))]
-    #[tokio::test(flavor = "current_thread")]
+    #[xmtp_common::test]
```

Comment:
the flavor should still work,
```suggestion
    #[xmtp_common::test(flavor = "current_thread")]
```